terraform/Makefile

```diff
@@ -8,7 +8,7 @@ TFDIR ?= example
 
 ADDFLAGS ?=
 BUILDFLAGS ?= $(ADDFLAGS) -ldflags '-w -s'
-CGOFLAG ?= CGO_ENABLED=1
+CGOFLAG ?= CGO_ENABLED=0
```

Comment:
You could just remove the `CGOFLAG`, unless you want it to be explicit.

Reply:
I can remove the line entirely and be explicit in the build command, like so:

```
GOOS=$(OS) GOARCH=$(ARCH) CGO_ENABLED=0 go build -o $(BUILDDIR)/terraform-provider-teleport $(BUILDFLAGS)
```

Was this your suggestion?